frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -198,6 +199,21 @@ export const getEventSourceData = (source: string) => {
   return eventSourceData[source];
 };
 
+export const getKameletSourceData = (kameletData?: K8sResourceKind) => {
+  return {
+    source: {
+      ref: {
+        apiVersion:
+          kameletData?.apiVersion ||
+          `${CamelKameletBindingModel.apiGroup}/${CamelKameletBindingModel.apiVersion}`,
```

Comment:
it'll always be `kameletData` version, we din't need or here and kameletData should be required field?